README.md

```diff
@@ -42,3 +42,24 @@ xarm-commander -h
 ```
 > :bulb: Can be also used for subcommands: `xarm-commander get_version -h`  
 
+## Configuration Parameters
+### Formatter:
+Formatting engine: `clangFormat`  
+Formatting style: `Visual Studio`
+
+### Linter:
```

Comment:
What is the actual command you are running on the console? (I.e., how do I reproduce your clang-tidy execution?

Reply:
I use an vscode extension, so it's defined in my settings..
But I guess you could use it by calling: `clang-tidy src/xarm-commander.cpp -checks="LIST_ALL_THE_CHECKS_HERE" -- -Iinclude`
or creating an .clang-tidy file..but I haven't used the file option yet..

Reply:
Can you please put the concrete command here? Not everyone is using VS Code...
Also for the formatting. I want the codebase to be editor-agnostic.

Reply:
Ah okay, sure!
I'll probably switch to Google then, or do you have a preference on formatting?